src/lib/helpers/config.ts

```diff
@@ -17,31 +19,71 @@ export const ConfigSchema = z.object({
         ),
     }).strict().default({}),
     cache: z.object({
-        enabled: z.boolean().default(true),
-        directory: z.string().default("/var/tmp"),
+        enabled: z.boolean().default(
+            Deno.env.get("CACHE_ENABLED") === "true"
+                ? true
+                : Deno.env.get("CACHE_ENALBED") === "false"
+                ? false
+                : true,
+        ),
+        directory: z.string().default(
+            Deno.env.get("CACHE_DIRECTORY") || "/var/tmp",
+        ),
     }).strict().default({}),
     networking: z.object({
-        ump: z.boolean().default(false),
+        ump: z.boolean().default(
+            Deno.env.get("NETWORKING_UMP") === "true" || false,
+        ),
         proxy: z.string().nullable().default(Deno.env.get("PROXY") || null),
         fetch: z.object({
-            timeout_ms: z.number().optional(),
+            timeout_ms: z.number().optional().default(
+                Number(Deno.env.get("NETWORKING_FETCH_TIMEOUT_MS") || null),
+            ),
             retry: z.object({
-                enabled: z.boolean(),
-                times: z.number().optional(),
-                initial_debounce: z.number().optional(),
-                debounce_multiplier: z.number().optional(),
+                enabled: z.boolean().default(
+                    Deno.env.get("NETWORKING_FETCH_RETRY_ENABLED") === "true" ||
+                        false,
+                ),
+                times: z.number().optional().default(
+                    Number(Deno.env.get("NETWORKING_FETCH_RETRY_TIMES")) || 1,
+                ),
+                initial_debounce: z.number().optional().default(
+                    Number(
+                        Deno.env.get("NETWORKING_FETCH_RETRY_INITIAL_DEBOUNCE"),
+                    ) || 0,
+                ),
+                debounce_multiplier: z.number().optional().default(
+                    Number(
+                        Deno.env.get(
+                            "NETWORKING_FETCH_RETRY_DEBOUNCE_MULTIPLIER",
+                        ),
+                    ) || 0,
+                ),
             }).strict().optional(),
-        }).strict().optional(),
+        }).strict().default({}),
     }).strict().default({}),
     jobs: z.object({
         youtube_session: z.object({
-            po_token_enabled: z.boolean().default(true),
-            frequency: z.string().default("*/5 * * * *"),
+            po_token_enabled: z.boolean().default(
+                Deno.env.get("JOBS_YOUTUBE_SESSION_PO_TOKEN_ENABLED") === "true"
+                    ? true
+                    : Deno.env.get("JOBS_YOUTUBE_SESSION_PO_TOKEN_ENABLED") ===
+                            "false"
+                    ? false
+                    : true,
```

Comment:
This should be the same as:
```suggestion
                    Deno.env.get("JOBS_YOUTUBE_SESSION_PO_TOKEN_ENABLED") === "false" ? false : true,
```
right? I think it's clearer to me this way - if it's explicitly `"false"`, then it's `false`, otherwise it's `true`.